spec/factories/games.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :scheduled_game, class: Game do
+    association :away_team, factory: :yankees
```

Comment:
This may have to be `Mets` instead of `Yankees` based on some of the FactoryGirl association problems we were seeing during development.